index.html

```diff
@@ -23,16 +24,209 @@ <h1>Hey there. I'm Emmanuel <br>
+                        <button class="button" type="button">See Project</button>
+                    </div>
+                    
+                </div>
+            </div>
+            <div class="cards">
+                <div class="card"></div>
+    
+                <div class="card-details">
+                    <h3>Multi-Post Stories <br> Gain+Glory</h3>
+    
+                    <ul class="card-tag" >
+                        <li class="tags ruby">Ruby on rails</li>
+                        <li class="tags css">css</li>
+                        <li class="tags jav">JavaScript</li>
+                        <li class="tags htm">html</li>
+                    </ul>
+    
+                    <div class="btn">
+                        <button class="button" type="button">See Project</button>
+                    </div>
+                    
+                </div>
+            </div>
+            <div class="cards">
+                <div class="card"></div>
+    
+                <div class="card-details">
+                    <h3>Multi-Post Stories <br> Gain+Glory</h3>
+    
+                    <ul class="card-tag" >
+                        <li class="tags ruby">Ruby on rails</li>
+                        <li class="tags css">css</li>
+                        <li class="tags jav">JavaScript</li>
+                        <li class="tags htm">html</li>
+                    </ul>
+    
+                    <div class="btn">
+                        <button class="button" type="button">See Project</button>
+                    </div>
+                    
+                </div>
+            </div>
+            <div class="cards">
+                <div class="card"></div>
+    
+                <div class="card-details">
+                    <h3>Multi-Post Stories <br> Gain+Glory</h3>
+    
+                    <ul class="card-tag" >
+                        <li class="tags ruby">Ruby on rails</li>
+                        <li class="tags css">css</li>
+                        <li class="tags jav">JavaScript</li>
+                        <li class="tags htm">html</li>
+                    </ul>
+    
+                    <div class="btn">
+                        <button class="button" type="button">See Project</button>
+                    </div>
+                    
+                </div>
+            </div>
+            <div class="cards">
+            
+    
+                <div class="card-details">
+                    <h3>Multi-Post Stories <br> Gain+Glory</h3>
+    
+                    <ul class="card-tag" >
+                        <li class="tags ruby">Ruby on rails</li>
+                        <li class="tags css">css</li>
+                        <li class="tags jav">JavaScript</li>
+                        <li class="tags htm">html</li>
+                    </ul>
+    
+                    <div class="btn">
+                        <button class="button" type="button">See Project</button>
+                    </div>
```

Comment:
- [ ] Kindly follow the button styling: Active, Hover, and Pressed (see design).
### Design
![figma](https://user-images.githubusercontent.com/81727988/176996634-00a583fd-5633-4e3c-a3c7-4bf5c029c89d.png)

Reply:
I have fix this error